actual/database.py

```diff
@@ -571,7 +574,14 @@ class ZeroBudgets(SQLModel, table=True):
 
```

Comment:
If the `ZeroBudgets` can be edited, it must inherit from the `BaseModel`, not `SQLModel`